benchmark/python/sparse/sparse_end2end.py

```diff
@@ -48,6 +49,12 @@
                     help='whether to call update_metric')
 parser.add_argument('--enable-logging-for', default="0",
                     help="Enable logging for the specified list of workers")
+parser.add_argument('--io-only', action='store_true',
```

Comment:
I actually prefer one option for `measure_only = IO/COMP/COMM, default to None`

Reply:
Changed.